float32 A
float32 B
float32 C
geometry_msgs/Point p_a
geometry_msgs/Point p_b
int32 ID
float32 length
float32 slope
